Modules/Microsoft365DSC/DSCResources/MSFT_IntuneVPNConfigurationPolicyIOS/MSFT_IntuneVPNConfigurationPolicyIOS.schema.mof

```diff
@@ -0,0 +1,101 @@
+    [Write, Description("The group Display Name that is the target of the assignment.")] String groupDisplayName;
+    [Write, Description("The collection Id that is the target of the assignment.(ConfigMgr)")] String collectionId;
+};
+[ClassVersion("1.0.0.0")]
+class MSFT_DeviceManagementConfigurationPolicyVpnOnDemandRule
+{
+    [Write, Description("Network Service Set Identifiers (SSIDs).")] String ssids[];
+    [Write, Description("DNS Search Domains.")] String dnsSearchDomains[];
+    [Write, Description("A URL to probe. If this URL is successfully fetched, returning a 200 HTTP status code, without redirection, this rule matches.")] String probeUrl;
+    [Write, Description("Action. Possible values are: connect, evaluateConnection, ignore, disconnect."), ValueMap{"connect", "evaluateConnection", "ignore", "disconnect"}, Values{"connect", "evaluateConnection", "ignore", "disconnect"}] String action;
+    [Write, Description("Domain Action, Only applicable when Action is evaluate connection. Possible values are: connectIfNeeded, neverConnect."), ValueMap{"connectIfNeeded", "neverConnect"}, Values{"connectIfNeeded", "neverConnect"}] String domainAction;
+    [Write, Description("Domains, Only applicable when Action is evaluate connection.")] String domains[];
+    [Write, Description("Probe Required URL. Only applicable when Action is evaluate connection and DomainAction is connectIfNeeded.")] String probeRequiredUrl;
+    [Write, Description("Network interface to trigger VPN. Possible values are: notConfigured, ethernet, wiFi, cellular."), ValueMap{"notConfigured", "ethernet", "wiFi", "cellular"}, Values{"notConfigured", "ethernet", "wiFi", "cellular"}] String interfaceTypeMatch;
+    [Write, Description("DNS Search Server Address.")] String dnsServerAddressMatch[];
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphVpnServer
+{
+    [Write, Description("Address (IP address, FQDN or URL)")] String address;
+    [Write, Description("Description.")] String description;
+    [Write, Description("Default server.")] Boolean isDefaultServer;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftvpnProxyServer
+{
+    [Write, Description("Proxy's automatic configuration script url.")] String automaticConfigurationScriptUrl;
+    [Write, Description("Address.")] String address;
+    [Write, Description("Port. Valid values 0 to 65535.")] uint32 port;
+};
+[ClassVersion("1.0.0")]
+class MSFT_targetedMobileApps
+{
+    [Write, Description("The application name.")] String address;
+    [Write, Description("The publisher of the application.")] String publisher;
+    [Write, Description("The Store URL of the application.")] String appStoreUrl;
+    [Write, Description("The application or bundle identifier of the application.")] String appId;
+};
+class MSFT_CustomData
+{
+    [Write, Description("Key for the custom data entry.")] String key;
+    [Write, Description("Value for the custom data entry.")] String value;
+};
+class MSFT_customKeyValueData
```

Comment:
Is this CIM instance required since there's already one defined above?

Reply:
If I'm understanding correctly, then yes. There is customData and customKeyValueData, they're different. 

 I've edited the description to reflect that one is a key,value pair and one is a name,value pair.

Reply:
let me know if I'm misunderstanding what you're saying?